### PR TITLE
Removing GRAIL references

### PR DESCRIPTION
**Pull request by rj3d**
@siddharthab could you look this over?

Engineering leadership and legal here at GRAIL have given us the okay to transfer ownership of the repository, but they want all references to GRAIL scrubbed first. Could you look this over to me to make sure everything looks good? Tests are currently broken for unrelated reasons, and I don't have the bandwidth to fix them at the moment. Can you run any tests that you'd like to make sure I didn't break anything?

### CODE REVIEW

**Comment by siddharthab** [comment]
Thank you! I can take care of the remaining work for this. Is there going to be some similar effort for the other bazel-* repos in the grailbio org?

**Comment by rj3d** [comment]
@siddharthab do you know if I can safely rename the importpath in [gazelle/BUILD](https://github.com/grailbio/rules_r/blob/grail-rename/gazelle/BUILD)? If so, do you have any suggestions as to what it could be renamed? Thanks!

**Comment by siddharthab** [comment]
> @siddharthab do you know if I can safely rename the importpath in [gazelle/BUILD](https://github.com/grailbio/rules_r/blob/grail-rename/gazelle/BUILD)? If so, do you have any suggestions as to what it could be renamed? Thanks!

Yes, you can rename it to `bazel.io/gazelle/rlang`. It will not be a breaking change.

**Comment by rj3d** [comment]
@siddharthab awesome, thank you! 

I hadn't heard any discussion of transferring the other bazel-* repos in the grailbio org. Would you mind listing out all of the ones that you think would be useful to transfer, and I can ask to confirm that it would be okay?

File: R/container/README.md [hunk 1]
```diff
@@ -33,22 +32,23 @@ container_pull(
...
+
 ## r_library_image
 
 In a `BUILD` file, define your library of R packages and install them
 in a Docker image using this rule. Dependencies are installed implicitly
 in an efficient layered mechanism.
 
 ```python
-load("@com_grail_rules_r//R:defs.bzl", "r_library")
+load("@com_rules_r//R:defs.bzl", "r_library")
```
Comment (siddharthab) [inline R/container/README.md]: Can we rename these to `rules_r` instead?

File: LANGUAGE.md
Comment (siddharthab) [inline LANGUAGE.md:1]: Does this file need to be deleted? We can remove the top paragraph. As far as I know, this is the last remaining source of this file.

**Comment by rj3d** [comment]
> Yes, you can rename it to `bazel.io/gazelle/rlang`. It will not be a breaking change.

Done! Thank you.

File: tests/container/BUILD [hunk 2]
```diff
@@ -22,17 +22,17 @@ r_library_tar(
         "tools",
     ],
     library = "//:library",
-    library_path = "grail/r-libs",
+    library_path = "R/r-libs",
```
Comment (siddharthab) [inline tests/container/BUILD]: Can you rename this to `rules_r` instead of just R?

File: tests/container/BUILD [hunk 3]
```diff
@@ -22,17 +22,17 @@ r_library_tar(
...
 ## Image related tests
 
-load("@com_grail_rules_r//R/container:defs.bzl", "r_binary_image", "r_library_image")
+load("@com_rules_r//R/container:defs.bzl", "r_binary_image", "r_library_image")
 
 r_library_image(
     name = "library_image",
     base = "@r_base//image",
-    directory = "grail",
+    directory = "r",
```
Comment (siddharthab) [inline tests/container/BUILD]: Same here. `rules_r` please instead of `r`.

File: README.md [hunk 4]
```diff
@@ -41,10 +45,11 @@ Bazel, over a custom solution of tracking the package dependency graph and
 triggering builds accordingly on each commit, is that R packages can be built
 and tested as part of one build system in multi-language monorepos.
 
-These rules are mature for production use. We use these rules internally at
+These rules are mature for production use. These rules are used at
```
Comment (siddharthab) [inline README.md]: If you want, we can remove the second sentence here completely.

**Review by siddharthab — CHANGES_REQUESTED** [review]

**Comment by siddharthab** [comment]
> Would you mind listing out all of the ones.

Sure. Thank you. The others are:
https://github.com/grailbio/bazel-toolchain
https://github.com/grailbio/bazel-compilation-database

Once approved, I can repeat the work you did for this repo and send you a review, so that it's easier for you.

**Comment by siddharthab** [comment]
If you want, I can take over this PR and finish addressing all the comments I left and fix the tests.

**Comment by rj3d** [comment]
> If you want, I can take over this PR and finish addressing all the comments I left and fix the tests.

That would be great! Is there anything you need from me too accomplish that?

File: R/container/README.md (hunk 1 above)
Comment (rj3d) [reply R/container/README.md]: That's fine with me!

File: LANGUAGE.md
Comment (rj3d) [reply LANGUAGE.md:1]: No reason it all has to be deleted. I added it back and removed the top paragraph.

File: tests/container/BUILD (hunk 3 above)
Comment (rj3d) [reply tests/container/BUILD]: Done!

**Comment by rj3d** [comment]
> 
> Sure. Thank you. The others are: https://github.com/grailbio/bazel-toolchain https://github.com/grailbio/bazel-compilation-database
> 
> Once approved, I can repeat the work you did for this repo and send you a review, so that it's easier for you.

I just asked to see what management thinks. I'll let you know when I hear back.

**Review by siddharthab — APPROVED** [review]

**Comment by siddharthab** [comment]
The test failures are unrelated. I am going to merge your changes first and then work on fixing the test failures. Thank you so much for all your help!

**Pull request merged**